internal/config/librarian_config.go

```diff
@@ -28,14 +28,16 @@ const (
 type LibrarianConfig struct {
 	GlobalFilesAllowlist []*GlobalFile    `yaml:"global_files_allowlist"`
 	Libraries            []*LibraryConfig `yaml:"libraries"`
+	TagFormat            string           `yaml:"tag_format"`
```

Comment:
Should we change it to `GlobalTagFormat`?

Reply:
I'm not sure: in this case, it's the default for the repository as the per-library config can override it. "Global" sounds like it might be the value no matter what